Formula/alerter.rb

```diff
@@ -0,0 +1,25 @@
+class Alerter < Formula
+  desc "Send User Alert Notification on macOS from the command-line"
+  homepage "https://github.com/vjeantet/alerter"
+  url "https://github.com/vjeantet/alerter/archive/refs/tags/004.tar.gz"
+  version "4"
```

Comment:
Actually, sorry, just noticed this. Do we need to explicitly specify the version like this? Isn't the version `004`?

Reply:
Look at [alerter's releases](https://github.com/vjeantet/alerter/releases), I assumed this was a quirky version of semantic versioning.

Reply:
The tags all specify `00x`, so let's keep that for the version. We can just omit the `version` line I think.

Reply:
I'm confused. I though version was necessary. I used `brew create` and I thought it failed to parse the version, and that's why it wasn't included. Should I change or remove version?

Reply:
Oh it *inferred* the version from the url. My bad